### DISCUSSION TITLE
Linking two servers

### DISCUSSION THREAD

**Discussion by adevoss** [inspircd/inspircd#2021] [Support]
After installing and configuring the first server I configured a second server (different ip and hostname, same subnet)
Both have local mysql db's for anope.

How do I know if those servers are linked? Both have these lines in the config file. With `netstat` I see both have port 7001 open.
Nothing in log files. The link to 127.0.0.1 for anope is shown in log.
```
<module name="spanningtree">
<link name="otherserver.xx.lan"
     ipaddr="<otherserverip>"
     port="7001"
     sendpass="password"
     recvpass="password">
<bind address="thisserverip" port="7001" type="servers">
```
The goal of linking is redundancy. Users connect to server one or server two. The servers have the same anope data so the same registered users and channels.
At the moment the db on server one contains a registered user. The db on server two is empty.

Get those two db's synced when the servers are linked? Or should I take another approach?

**Comment by sadiepowell** [comment]
Services data is not synchronised over the network. You can only have one services server on your network. If you're using the MySQL Anope backend you can do failover with [MySQL replication](https://dev.mysql.com/doc/refman/8.0/en/replication.html).

You should ask future Anope questions on irc.anope.org #anope or on [the Anope forum](https://forum.anope.org/).

**Reply by adevoss** [reply]
Thanks for your reply. Now I know how a IRC network looks like. That information I couldn't find online.
